.github/ISSUE_TEMPLATE/broken-website.md

```diff
@@ -1,60 +1,57 @@
 ---
-name: Broken website
-about: Creates a broken website report about Dark Reader
-title: "[Broken Website] Replace with title"
+name: Broken Website Report
+about: Create a report about a website issue you experienced while using Dark Reader.
+title: "[Broken Website] Place title here"
```

Comment:
We actually had this on before-hand but changed to `Replace with title` as it was thought you would place the title after `place title here` So I would often see `[Broken Website] Place title here *Actually title*`